package.json

```diff
@@ -6,9 +6,9 @@
   "scripts": {
     "start": "nodemon lib/index.js --exec babel-node",
     "build": "babel lib -d dist",
-    "lint": "eslint lib/",
+    "lint": "./node_modules/.bin/eslint lib/",
```

Comment:
It didn't work for me before I did that, but now I revert it it works! I am thoroughly confused.

Thanks for the tip :)